src/utils/date-utils.js

```diff
@@ -0,0 +1,9 @@
+const getFormattedDate = () => {
```

Comment:
Add a comment which explains what does this method do. thanks!

Reply:
Yes @gsluthra added the comments.